### PR TITLE
Update "How we work" page

### PR DESCRIPTION
**Pull request by maya**
Fixes issue(s) #2749. Related: https://github.com/18F/design-lab/issues/18

[CircleCI](https://circleci.com/gh/18F/18f.gsa.gov/tree/update-how-we-work)

[:sunglasses: PREVIEW](https://federalist-proxy.app.cloud.gov/preview/18f/18f.gsa.gov/update-how-we-work/how-we-work/)

[Preview README for this branch](https://github.com/18F/18f.gsa.gov/blob/BRANCH_NAME/README.md)

Changes proposed in this pull request:
-
-
-

Checklist:
- [ ] All images being added have been optimized **_--> [learn more](18f.gsa.gov/styleguide/images/#using-and-optimizing-jpg-and-png) about how to optimize images <--_**

/cc @relevant-people

### CODE REVIEW

File: _sass/_templates/how-we-work.scss [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        padding-top: 3rem;
+
+        @include media($medium-screen) {
+          padding-top: 0;
+        }
+      }
+    }
+
+    img {
+      width: max-content;
```
[the remark this answers is not in the document]
Comment (maya) [reply _sass/_templates/how-we-work.scss]: Fixed in 048aa4c.

File: _sass/_templates/how-we-work.scss [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+.page-how-we-work {
+  .graphic-list {
+    list-style: none;
+    max-width: 100%;
+    padding-left: 0;
+
+    li {
+      display: inline-block;
+      font-weight: $font-bold;
+      width: 48%;
```
[the remark this answers is not in the document]
Comment (maya) [reply _sass/_templates/how-we-work.scss]: Fixed in 048aa4c.

**Comment by maya** [comment]
Thanks for reviewing, @jenniferthibault - fixed in 240b064.

**Comment by maya** [comment]
Design updates are in.

**Comment by maya** [comment]
Using the thin gray line border as you suggested @jenniferthibault, though I noticed a thick blue line in the page here: https://18f.gsa.gov/what-we-deliver/data-act/

screen shot 2018-10-24 at 2 58 38 pm

**Comment by eddietejeda** [comment]
I suggest we keep it consistent and switch that page over to use the thin line.

**Comment by jenniferthibault** [comment]
OOh good spotting @maya I hadn't found that one. If it's all the same @eddietejeda, I think the thicker blue line that's already in place on the case studies page feels better, so would rather use that as the canonical pattern to stick with. (And I'll add that into the design files as well so we can find it again)

**Pull request merged**